webExtensionApis/study/src/studyUtils.js

```diff
@@ -55,15 +54,12 @@ const { TelemetryEnvironment } = Cu.import(
 *      (used to capture feature-specific state) sent as Map(string,string).
 *    - "shield-study-error": data used to notify, group and count some kinds
 *      of errors from shield studies
-*  - ShieldUtils API ducktypes:
-*    - "weightedVariations": the array of branch name:weight pairs used to
-*      randomly assign the user
-*    to a branch
-*    - "webExtensionMsg": the message object passed into the
-*      StudyUtils.respondToWebExtensionMessage method
-*    - "studySetup": the options object passed into the StudyUtils.setup method
 */
-import schemas from "./schemas";
+const schemas = {
```

Comment:
sgtm.  Thanks for fixing this.  I HATE the schemas, so Ideas are very welcome for keeping these in sync both places.